test/jasmine/tests/animate_test.js

```diff
@@ -720,7 +720,7 @@ describe('Animating multiple axes', function() {
         destroyGraphDiv();
     });
 
-    it('updates ranges of secondary axes', function(done) {
+    it('@flaky updates ranges of secondary axes', function(done) {
```

Comment:
Can you think of any other flaky tests worth adding to the list @alexcjohnson ?

Reply:
Not sure, I think you got the most common ones, but now it's super easy to add `@flaky` if another test fails spuriously. Lets just keep an eye out for failures over the next few days after merging this.